kratos/solving_strategies/convergencecriterias/residual_criteria.h

```diff
@@ -432,17 +431,17 @@ class ResidualCriteria
     ///@name Member Variables
     ///@{
 
-    TDataType mRatioTolerance;      /// The ratio threshold for the norm of the residual
+    TDataType mRatioTolerance = 0.0;      /// The ratio threshold for the norm of the residual
```

Comment:
Wouldn't it be better like the following?


```suggestion
    TDataType mRatioTolerance{};      /// The ratio threshold for the norm of the residual
```

Reply:
I can change that

Reply:
Okay now?